.eslintrc.js

```diff
@@ -26,6 +31,23 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'quotes': ['error', 'single'],
     'semi': ['error', 'always'],
+    'jsdoc/check-tag-names': [
+      'error',
+      {
+        'definedTags': [
+          'model',
+          'slot'
+        ]
+      }
+    ],
+    'jsdoc/no-undefined-types': [
+      'warn',
+      {
+        'definedTypes': [
+          'unknown'
```

Comment:
You were using `@return {unknown}` which is not a valid type. Adding this here prevents it from being marked as a linting warning.

Reply:
Yeah, I think I found that somewhere on the internet and copied it. But it doesnt look quite right. I think maybe this should just be `@return {*}`